packages/ripple-nuxt-tide/lib/module.js

```diff
@@ -2,12 +2,17 @@ import defaults from './config/defaults'
 import * as configLoader from './core/config-loader'
 import { RPL_HEADER } from './config/constants'
 import tideSearchApiMiddleware from '@dpc-sdp/ripple-tide-search-api'
+import logger from './core/logger'
 
 const path = require('path')
 
 const nuxtTide = function (moduleOptions) {
   const options = Object.assign(defaults, this.options.tide, moduleOptions)
 
+  if (!options || (options && !options.baseUrl)) {
```

Comment:
@rubyllamadora I think you can use `if (!(options.baseUrl)) {`. Because `options` should be defined as object already in line 10.

Reply:
✔️ Will update.